public/styles/style-quiz.css

```diff
@@ -355,4 +355,13 @@ footer p {
         text-align: center;
     }
     
-}
\ No newline at end of file
+}
+
+a:focus {
+    outline: 2px solid var(--bs-secondary-bg);
+}
+
+.navbar-brand img:focus {
+    outline: none !important;
+    box-shadow: 0 0 2px 2px var(--bs-secondary-bg);
+}  
```

Comment:
Is not necessary, at least on chrome and firefox. It works simply with the code above, for the a-tag alone. But it really does depend on the browser sometimes. But so i'd leave this off